src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java

```diff
@@ -0,0 +1,175 @@
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.api.ArticleContent;
+import com.wikia.webdriver.common.core.api.UserRegistration;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.SignUpUser;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.pages.ArticlePage;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.global_navitagtion.NavigationBar;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.minieditor.MiniEditorComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.DetachedSignInPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.SignInPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.messagewall.MessageWall;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialWikiActivityPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.anonymization.SpecialAnonymizationUserPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool.ConfirmationModalPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool.HelpPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool.SpecialRenameUserPage;
+import com.wikia.webdriver.testcases.articlecrudtests.ArticleCRUDUserTests;
+
+import jdk.nashorn.internal.runtime.regexp.RegExp;
+import org.assertj.core.util.Compatibility;
+import org.joda.time.DateTime;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.testng.annotations.Test;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.UUID;
+
+@SuppressWarnings("SpellCheckingInspection")
+@InBrowser(browser = Browser.CHROME)
+@Execute(onWikia = "communitytest")
+@Test(groups = "anonymization")
+public class AnonymizationTests extends NewTestTemplate {
+
+  private SpecialAnonymizationUserPage openAnonymizationPage(User user) {
+    WikiBasePageObject base = new WikiBasePageObject();
+    base.loginAs(user);
+    return new SpecialAnonymizationUserPage().open();
+  }
+
+  private static final String ERROR_MESSAGE =
+      "We don't recognize these credentials. Try again or register a new account.";
+
+  @Test
+  public void anonymizedUserCannotLogin() {
+
+    Credentials credentials = new Credentials();
+    String timestamp = Long.toString(DateTime.now().getMillis());
+    String qanon = "QAanon" + timestamp;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "QAanon" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)